packages/tools/kadena-cli/src/devnet/faucet/deploy/faucet.ts

```diff
@@ -0,0 +1,72 @@
+import type { ICommandResult } from '@kadena/client';
+import {
+  Pact,
+  createClient,
+  createSignWithKeypair,
+  isSignedTransaction,
+} from '@kadena/client';
+import type { ChainId } from '@kadena/types';
+
+import { DOMAIN, NETWORK_ID, SENDER_00 } from './constants.js';
+
+import fs from 'node:fs';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+export const deployFaucet = async ({
+  chainId,
+  upgrade,
+  namespace,
+}: {
+  chainId: ChainId;
+  upgrade: boolean;
+  namespace: string;
+}): Promise<Record<string, ICommandResult>> => {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const __filename = fileURLToPath(import.meta.url);
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  const __dirname = path.dirname(__filename);
+  const faucetContract = fs.readFileSync(
```

Comment:
this won't work when we publish. Please just put the contract in a `.ts` file and do 
```
export const contract = `
<contract>
`;
```